### PR TITLE
Add periodic update action, cleanup fetch.sh

### PR DESCRIPTION
**Pull request by ix5**
Add a GitHub Actions file to update the data every day at ~~midnight~~ 03:47.

Use arguments for fetch.sh instead of hardcoding the path.

### CODE REVIEW

**Comment by foosint** [comment]
There is one downside with scheduled runs though. There is no guarantee that the workflow will run at the exact scheduled time. Delays of 1h or more are possible.

**Comment by ix5** [comment]
> There is one downside with scheduled runs though. There is no guarantee that the workflow will run at the exact scheduled time. Delays of 1h or more are possible.

Wasn't aware of that, thanks. Found [this discussion](https://github.com/orgs/community/discussions/27130) for details.

It seems one should avoid scheduling a script at peak times, then. I still think running this thing on a schedule is the right approach rather than relying on individual contributors.
Otherwise, we'll just create 2 repos: One for @fdov21 to push to manually, and one which archives things via GH actions as a secondary backup.

**Comment by fdov21** [comment]
I don't think it matters if the action runs at 0000 or 0100, does it?

**Comment by fdov21** [comment]
Merged.

I've disabled my git add and push locally, for now.
Still fetching just to have a backup.

File: push.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+BRANCH=master
+REMOTE=origin
```
Comment (ix5) [inline push.sh:3]: This should also be:
```diff
+FOLDER=$1
+EXPORT_URL='https://www.google.com/maps/d/u/0/kml?mid=180u1IkUjtjpdJWnIC0AxTKSiqK4G6Pez'
+SAVE_TO=${FOLDER:-.}/$(date '+%y%m%d_%H%M').kmz
```
Maybe you can push that quickly

Comment (fdov21) [reply push.sh:3]: Thanks. Should be updated now.
My fat-finger conflict resolving broke this.

**Pull request merged**